runner/src/main.rs

```diff
@@ -73,10 +74,74 @@ fn build_examples(opt: &BuildExamples) -> Step {
     }
 }
 
+fn run_examples(opt: &RunExamples) -> Step {
+    match opt.application_variant.as_str() {
+        "rust" => Step::Multiple {
+            name: "examples".to_string(),
+            /// TODO(#396): Check that all the example folders are covered by an entry here, or
+            /// explicitly ignored. This will probably require pulling out the `Vec<Example>` to a
+            /// top-level method first.
+            steps: vec![
+                Example {
+                    name: "abitest".to_string(),
+                    rust_module_names: vec!["module_0".to_string(), "module_1".to_string()],
+                    // TODO(#730): reinstate Roughtime tests when Rust runtime supports them.
+                    // TODO(#1040): reinstate storage tests when Rust runtime supports them.
+                    // TODO(#954): reinstate gRPC client server-streaming tests when Rust runtime supports them.
+                    // TODO(#953): reinstate gRPC server server-streaming tests when Rust runtime supports them.
+                    additional_client_flags: vec![
+                        "--cert_chain=../../../../../../../../examples/certs/local/local.pem"
```

Comment:
Does rust have a way of specifying paths from the project root?

Reply:
This is not a Rust thing, it's bazel that runs this binary in a weird PWD. But maybe we can figure out the project root from rust and then just pass an absolute path here, is that what you meant?

Reply:
Thanks for explaining, that makes sense. :) 

Yeah, I was basically wondering if there was some way to avoid the `../../../../../../../` syntax. I personally find it a bit hard to read and as-far-as-understand it needs to renaming when moving this file. 

I'm more like curious though, it's not a big issue that necessarily needs adressing.